CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Compatibility Note
```

Comment:
This is not a header from https://keepachangelog.com/en/1.0.0/. We have historically just put general release notes under the `## [Release]` header, I think this can just be dropped.